lib/error_tracker.ex

```diff
@@ -300,8 +328,16 @@ defmodule ErrorTracker do
   end
 
   defp upsert_error!(error, stacktrace, context, breadcrumbs, reason) do
-    existing_status =
-      Repo.one(from e in Error, where: [fingerprint: ^error.fingerprint], select: e.status)
+    status_and_muted_query =
+      from e in Error,
+        where: [fingerprint: ^error.fingerprint],
+        select: {e.status, e.muted}
+
+    {existing_status, muted} =
+      case Repo.one(status_and_muted_query) do
+        {existing_status, muted} -> {existing_status, muted}
+        nil -> {nil, nil}
```

Comment:
It works as it is, but to keep type consistency it would be:
```suggestion
        nil -> {nil, false}
```

Reply:
Good catch! Fixed!